#--- slim-1.3.2/Makefile~	2011-07-12 23:56:02.000000000 +0300
#+++ slim-1.3.2/Makefile	2011-07-12 23:57:17.805261930 +0300
#@@ -5,9 +5,9 @@
# #######################################################
# CXX=/usr/bin/g++
# CC=/usr/bin/gcc
#-CFLAGS=-Wall -I. -I/usr/include/freetype2 -I/usr/include/freetype2/config -I/usr/include/libpng12 -I/usr/include
#+override CFLAGS += -I. -I/usr/include/freetype2 -I/usr/include/freetype2/config -I/usr/include/libpng -I/usr/include
# CXXFLAGS=$(CFLAGS)
#-LDFLAGS=-lXft -lX11 -lfreetype -lXrender -lfontconfig -lpng12 -lz -lm -lcrypt -lXmu -lpng -ljpeg -lrt
#+override LDFLAGS += -lXft -lX11 -lfreetype -lXrender -lfontconfig -lz -lm -lcrypt -lXmu -lpng -ljpeg -lrt
# CUSTOM=-DHAVE_SHADOW
# ifdef USE_PAM
# LDFLAGS+= -lpam
#@@ -33,7 +33,7 @@
# all: slim
# 
# slim: $(OBJECTS)
#-	$(CXX) $(LDFLAGS) $(OBJECTS) -o $(NAME)
#+	$(CXX) $(OBJECTS) -o $(NAME) $(LDFLAGS) 
# 
# .cpp.o:
# 	$(CXX) $(CXXFLAGS) $(DEFINES) $(CUSTOM) -c $< -o $@
--- slim-1.3.3/CMakeLists.txt~	2012-02-22 01:59:52.000000000 +0200
+++ slim-1.3.3/CMakeLists.txt	2012-03-24 12:12:56.665259427 +0200
@@ -23,7 +23,7 @@
 
 set(CMAKE_INSTALL_PREFIX "/usr/local" CACHE PATH "Installation Directory")
 set(PKGDATADIR "${CMAKE_INSTALL_PREFIX}/share/slim")
-set(SYSCONFDIR "/etc")
+set(SYSCONFDIR "${SYSCONF_INSTALL_DIR}")
 set(MANDIR "${CMAKE_INSTALL_PREFIX}/share/man")
 
 set(SLIM_DEFINITIONS)
